src/components/SharedComponents/SwitchComponent.style.tsx

```diff
@@ -7,24 +7,24 @@ export const SwitchLabel = styled(Grid)`
 `;
 
 export const SwitchGrid = styled(Grid)<{
-  viewAllCounties: boolean | undefined;
-  isModal?: boolean;
+  $checked: boolean | undefined;
+  $isModal?: boolean;
```

Comment:
Made these into [transient props](https://styled-components.com/docs/api#transient-props) to avoid a React warning about invalid props being passed to a DOM component.